.goreleaser.yml

```
@@ -525,6 +525,8 @@ docker_manifests:
 checksum:
   name_template: "{{ .ProjectName }}-v{{ .Version }}-SHA256SUMS"
   algorithm: sha256
+  extra_files:
+    - glob: "./observiq-otel-collector.msi*"
```

I think we should probably checksum the artifacts archive

```suggestion
  extra_files:
    - glob: "./observiq-otel-collector.msi*"
    - glob: "observiq-otel-collector-${{ github.ref_name }}-artifacts.tar.gz"
```

---

We can't because the archive contains the checksum. That's why we build it after the fact. It's used for offline agent updates.

---

Of course!